<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GoProClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>GoPro Network</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Please select from the available networks:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="listView"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>All Selected Cameras:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="startRecBut">
              <property name="text">
               <string>Start Recording</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stopRecBut">
              <property name="text">
               <string>Stop Recording</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="verifyBut">
              <property name="text">
               <string>Verify Connection</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Command/History Log:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="history"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Individual Camera Status (Ordered By Selection):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
              <horstretch>57</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>700</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <column>
             <property name="text">
              <string>Network ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>GoPro Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Connected?</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Is Recording?</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ping Successful</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Live Stream</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>300</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Feature not yet supported.</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
